public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -39,6 +39,35 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 	</div>
 </div>
 
+<b>Connection Certificate Files</b>
+
+<div class="gf-form-group">
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">SSL Root Cert</span>
+ 		<input type="text" class="gf-form-input gf-form-input--has-help-icon" ng-model='ctrl.current.jsonData.sslrootcertfile' placeholder="SSL root cert file"></input>
+		<info-popover mode="right-absolute">
+			If selected SSL Mode needs a SSL Root Certificate, provide here the path to the file. 
+      Be sure that the file is readable by grafana user.
+		</info-popover>
+	</div>
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">SSL User Cert</span>
+ 		<input type="text" class="gf-form-input gf-form-input--has-help-icon" ng-model='ctrl.current.jsonData.sslcertfile' placeholder="SSL user cert file"></input>
+		<info-popover mode="right-absolute">
+			If selected SSL Mode needs a SSL User Certificate, provide here the path to the file. 
+      Be sure that the file is readable by grafana user.
+		</info-popover>
+	</div>
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">SSL User Key</span>
+ 		<input type="text" class="gf-form-input gf-form-input--has-help-icon" ng-model='ctrl.current.jsonData.sslkeyfile' placeholder="SSL user key file"></input>
+		<info-popover mode="right-absolute">
+			If selected SSL Mode needs a SSL User Key, provide here the path to the file. 
+      Be sure that the file is <b>only</b> readable by grafana user.
```

Comment:
```suggestion
      Be sure that the file is <b>only</b> readable by the grafana user.
```

Reply:
Do not use bold for emphasis. Use italic instead.

Reply:
@oddlittlebird I thought the same, but I'm too unfamiliar with the Grafana UI design to say.

Reply:
This is not a matter of UI, it is a matter of documentation. Specifically, it is covered in the Microsoft Style Guide, which is referenced in the [Grafana Documentation Style Guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md)

We bold things like button names, links, and other things on the screen that the user might click on. We italicize for emphasis. Using bold for emphasis can cause confusion.